### ISSUE TITLE
Accessibility - Insufficient color contrast

### ISSUE THREAD

**Issue by AmyJuneH** [badcamp/badcamp-2019#103]
On sponsor page and various other places where this styling is applied See screenshot

![Screen Shot 2019-08-16 at 7 02 14 AM](https://user-images.githubusercontent.com/20327232/63173635-25962380-bff5-11e9-85d6-7342dd66616b.png)

**Comment by AmyJuneH** [comment]
https://www.drupal.org/project/badcamp/issues/3076621
